### PR TITLE
Google Popup

### PR DESCRIPTION
**Pull request by denislukianenko**
### HTML & CSS practice: Google-style Popup

[Demo](https://denislukianenko.github.io/google-popup/) |
[Code base](https://github.com/denislukianenko/google-popup)

Tried to not to forget about accessibility: every element has ```:focus``` state, many have custom ```area-label```. Try to navigate with Tab :) You can also close popup by clicking somewhere outside of it.

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

File: submissions/denislukianenko/html-css-popup/style.css [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-weight: 100;
+}
+
+header {
```
Comment (A-Ostrovnyy) [inline submissions/denislukianenko/html-css-popup/style.css]: Avoid styling by tag.

File: submissions/denislukianenko/html-css-popup/style.css [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+.services-popup {
+  display: none;
+  position: absolute;
+}
+
+/** POPUP SWITCHING **/
+#popup-state-checkbox:checked + .popup-state-label + .services-popup {
+  display: block;
+}
+#popup-state-checkbox:checked + .popup-state-label > .big-closing-plate {
```
[the remark this answers is not in the document]
Comment (denislukianenko) [reply submissions/denislukianenko/html-css-popup/style.css]: Whoa, that's crazy!

File: submissions/denislukianenko/html-css-popup/index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          <label for="more-state-checkbox" class="more-services-button">
+            <span class="more">More</span>
+            <span class="less">Less</span>
+          </label>
+        </div>
+        <button
+          class="header-notifications-toggle"
+          aria-label="Notifications Settings Switch"
+        ></button>
+        <button class="header-avatar" ria-label="User Settings Switch"></button>
```
Comment (A-Ostrovnyy) [inline submissions/denislukianenko/html-css-popup/index.html]: aria-label

File: submissions/denislukianenko/html-css-popup/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (A-Ostrovnyy) [reply submissions/denislukianenko/html-css-popup/style.css]: It's only 3-5 selectors so this is not a big mess. If you like that code style you can leave this.

**Comment by A-Ostrovnyy** [comment]
Really good work.
Some className is too long as for me but if its good to you so it's ok.
See my comments above.

**Comment by A-Ostrovnyy** [comment]
Good job!

**Pull request merged**